lnwallet/interface_test.go

```diff
@@ -1269,32 +1269,33 @@ func testTransactionSubscriptions(miner *rpctest.Harness,
 		outputAmt = btcutil.SatoshiPerBitcoin
 		numTxns   = 3
 	)
-	unconfirmedNtfns := make(chan struct{})
+	errCh1 := make(chan error)
 	switch alice.BackEnd() {
 	case "neutrino":
 		// Neutrino doesn't listen for unconfirmed transactions.
 	default:
 		go func() {
+			defer close(errCh1)
```

Comment:
should use the same pattern both places. I would not close the channel, but instead explicitly send `nil` to indicate success.